client/extensions/woocommerce/lib/countries/index.js

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import CA from './CA';
+import US from './US';
+
+// Note: We are not using the other state name resources in calypso
```

Comment:
Shoot - I might rather use those. I searched calypso for "Manitoba" and, not finding it, created these instead. This should be an easy change. Checking that now.